### PR TITLE
Display announcements, update alert styles

### PR DESCRIPTION
**Pull request by sssoz**

### CODE REVIEW

File: templates/frontend/objects/announcement_summary.tpl [hunk 1]
```diff
@@ -23,9 +23,9 @@
 	<div class="announcement-summary-date">
 		{$announcement->getDatePosted()|date_format:$dateFormatLong}
 	</div>
-	<div class="announcement-summary-description">
+	<p class="announcement-summary-description">
 		{$announcement->getLocalizedDescriptionShort()|strip_unsafe_html}
-		<div class="announcement-summary-more">
+		<p class="announcement-summary-more">
```
Comment (NateWr) [inline templates/frontend/objects/announcement_summary.tpl]: A `<p>` should not be nested inside of another `<p>`. I'm also not sure if it's valid to nest a `<div>` inside of a `<p>`. You probably want the outer `<p>` to be  a `<div>`. Or to do something like this:

```
<div>
  <p>...description...</p>
  <p><a>...</a></p>
</div>
```

File: templates/frontend/pages/indexJournal.tpl [hunk 2]
```diff
@@ -89,6 +88,21 @@
 			</div>
 		{/if}
 
+		{if $announcements}
+		<section class="row homepage-announcements">
+			<h2 class="sr-only">{translate key="announcement.announcementsHome"}</h2>
+			{foreach from=$announcements item=announcement}
+				<article class="col-md-4 homepage-announcement">
+					<h3 class="homepage-announcement-title">{$announcement->getLocalizedTitle()|escape}</h3>
+					<p>{$announcement->getLocalizedDescriptionShort()|strip_unsafe_html} <br><a href="{url router=$smarty.const.ROUTE_PAGE page="announcement" op="view" path=$announcement->getId()}">{translate key="common.more"}…</a></p>
```
Comment (NateWr) [inline templates/frontend/pages/indexJournal.tpl]: The `...` here probably needs to be translatable for other languages. I'd suggest a theme-specific locale string like:

```
<message key="plugins.themes.healthSciences.more">{$text}...</message>
```

And then doing something like:

```
{capture var="more"}{translate key="common.more"}{/capture}
{translate key="plugins.themes.healthSciences.more" text=$more}
```

File: templates/frontend/objects/announcement_summary.tpl (hunk 1 above)
Comment (sssoz) [reply templates/frontend/objects/announcement_summary.tpl]: Oops! Thanks for catching that. I will fix it now.

File: templates/frontend/pages/indexJournal.tpl (hunk 2 above)
Comment (sssoz) [reply templates/frontend/pages/indexJournal.tpl]: rebased + force-pushed!

File: templates/frontend/objects/announcement_summary.tpl (hunk 1 above)
Comment (sssoz) [reply templates/frontend/objects/announcement_summary.tpl]: rebased + force-pushed!

**Comment by NateWr** [comment]
Merge when ready! :+1:

**Comment by Vitaliy-1** [comment]
@NateWr 
In general, do I need to make additional actions when merging a branch that has forced pushed commits?

**Comment by NateWr** [comment]
As long as there are no merge conflicts, and you see the commits listed here that you expect (and no others) then you can merge with confidence -- or if not confidence, then at least a faint gimmer of hope.

**Pull request merged**